pydantic/functional_validators.py

```diff
@@ -229,13 +223,6 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
             serialization = None
 
         input_schema = handler.generate_schema(self.json_schema_input_type)
```

Comment:
Note to self: although the default `json_schema_input_type` value is `Any`, we should also account for `PydanticUndefined` if explicitly set.